app/services/update_duplicate_matches.rb

```diff
@@ -0,0 +1,63 @@
+class UpdateDuplicateMatches
+  def initialize
+    @matches = GetFraudMatches.call
+  end
+
+  def save!
+    @matches.each do |match|
+      save_match(match)
+    end
+
+    message = <<~MSG
+      \n#{Rails.application.routes.url_helpers.support_interface_fraud_auditing_matches_url}
+      :face_with_monocle: There #{new_match_count == 1 ? 'is' : 'are'} #{new_match_count} new duplicate candidate #{'match'.pluralize(new_match_count)} today :face_with_monocle:
+      :gavel: #{fraudulent_match_count} #{'match'.pluralize(fraudulent_match_count)} #{fraudulent_match_count == 1 ? 'has' : 'have'} been marked as fraudulent :gavel:
+      :female-detective: In total there #{total_match_count == 1 ? 'is' : 'are'} #{total_match_count} #{'match'.pluralize(total_match_count)} :male-detective:
+    MSG
+
+    SlackNotificationWorker.perform_async(message)
+  end
+
+private
+
+  def save_match(match)
+    fraud_match, candidate = create_or_update_fraud_match(match)
+    notify_candidate(candidate, fraud_match)
+    candidate.update!(submission_blocked: true)
+  end
```

Comment:
Maybe wrap this in a transaction and send the email after?

```suggestion
fraud_match, candidate = nil

ActiveRecord::Base.transaction do
    fraud_match, candidate = create_or_update_fraud_match(match)
    candidate.update!(submission_blocked: true)
end

notify_candidate(candidate, fraud_match)
```